app.js

```diff
@@ -56,19 +57,19 @@ var previousValues = [];
 // grander callback that'll grab clicks as well as run the random image producer
 function handleClick(event) {
 
-  var currentTarget = event.target.currentSrc.slice(46);
 
-  for ( var i = 0; i < Item.catalog.length; i++) {
-    if (Item.catalog[i].filepath === currentTarget) {
+  for (var i in Item.catalog) {
```

Comment:
Cool!

Do you get the difference between this and a normal for loop? 

You can also use this syntax for what?